benchmarks/benchmark_latency.py

```diff
@@ -115,6 +116,11 @@ def run_to_completion(profile_dir: Optional[str] = None):
     parser.add_argument('--enforce-eager',
                         action='store_true',
                         help='enforce eager mode and disable CUDA graph')
+    parser.add_argument('--kv-cache-dtype',
+                        type=str,
+                        choices=['fp8', None],
+                        default=None,
+                        help='Data type for kv cache storage.')
```

Comment:
Please specify the default behavior here. And one question, why `'fp16'` is not a valid option here?

Reply:
Additionally, can we call `fp8` `fp8_e5m2` across this change? Because there are different ways to implement fp8.

Reply:
The default kv cache data type is same with the model dtype. So it may be not suitable to make `fp16` as default, as the model dtype will be float/bfloat16/half/...